flux-plugin/src/flux-runtime/runtime.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+Get all pods and then from them get the controllers for flux
+
+1. helm-controller
+2. kustomize-controller
+3. notification-controller
+4. source-controller
+5. image-reflector-controller
+6. image-automation-controller
+
+*/
+import { K8s } from '@kinvolk/headlamp-plugin/lib';
+import { Link, SectionBox, Table } from '@kinvolk/headlamp-plugin/lib/components/common';
+import { Link as MuiLink } from '@mui/material';
+
+export default function FluxRunTime(props) {
+  const [pods, error] = K8s.ResourceClasses.Pod.useList();
+  const [crds] = K8s.ResourceClasses.CustomResourceDefinition.useList();
+  const helmController = pods?.filter(pod => pod.metadata.labels['app'] === 'helm-controller');
+  const kustomizeController = pods?.filter(
+    pod => pod.metadata.labels['app'] === 'kustomize-controller'
+  );
+  const notificationController = pods?.filter(
+    pod => pod.metadata.labels['app'] === 'notification-controller'
+  );
+  const sourceController = pods?.filter(pod => pod.metadata.labels['app'] === 'source-controller');
+  const imageReflectorController = pods?.filter(
+    pod => pod.metadata.labels['app'] === 'image-reflector-controller'
+  );
+  const imageAutomationController = pods?.filter(
+    pod => pod.metadata.labels['app'] === 'image-automation-controller'
+  );
```

Comment:
We could add this filtering to the list query. But maybe now it doesn't matter so much, with react-query.